graph/src/components/store/mod.rs

```diff
@@ -833,7 +832,7 @@ pub enum UnfailOutcome {
 #[derive(Clone)]
 pub struct StoredDynamicDataSource {
     pub name: String,
-    pub address: Option<Address>,
+    pub param: Option<Box<[u8]>>,
```

Comment:
The `Box<[u8]>` could be `graph::data::store::scalar::Bytes` since that's used in lots of other places

Reply:
Done